### DISCUSSION TITLE
Trying to figure if a satellite is behind the earth from a location on earth

### DISCUSSION THREAD

**Discussion by sellick-phil** [skyfielders/python-skyfield#817] [Q&A]
I am trying to work out how to want to check from various points onthe earth the potential conjunction , between a LEO or MEO satellite and a GEO satellite, so we cycle through a time step loop to check the conjunction angles using 2 positions & working out the relative angle AOS = pos1.separation_from(pos2) (thanks very much to Brandon for this help) . 

However I think it is also seeing angles through the earth as when I insert a Satellite based in Australia, viewing & LEO from London, it still sees it . So we work out the posting of a satellite relative to earth  as below and then trying to check if the GEo is behind the earth or not using 

pos3 = (earth + geoSat).at(t)
pos4=pos3.is_behind_earth

Then we try and check if it is behind the earth using  () but it is always false (as I guess it doesn't have a relative position :-) ). If I try pos3 = (earth + usno).at(t).observe(geoSat)

Exception has occurred: ValueError you can only observe() a body whose vector center is the Solar System Barycenter, but this vector has the center 399 EARTH

So I guess I can only check for planets that are behind the earth not satellites ? Has anyone tried to check in a simple way if a satellite is behind the earth at a specific time ? I could check against the long & Lat but that might take a lot of itterations

Brilliant library , very very useful for our satellite projects amazing

**Comment by brandon-rhodes** [comment]
```
pos3 = (earth + geoSat).at(t)
```

This line of code computes the position of the satellite from the Solar System’s center. If you want its position from London, you can try:

```
pos3 = (geoSat - london).at(t)
pos3.is_behind_earth()
```

See whether that relative-to-London location gives you the result you want.

**Reply by sellick-phil** [reply]
Aaagghhh I have been so stupid, yes of course. Too focused on the wrong things ... thanks once again
